diff -ru pycairo-1.18.0/setup.py pycairo-1.18.0~/setup.py
--- pycairo-1.18.0/setup.py	2018-11-04 02:56:41.000000000 -0800
+++ pycairo-1.18.0~/setup.py	2019-01-08 17:20:44.546588072 -0800
@@ -326,10 +326,7 @@
             return
 
         if self.pkgconfigdir is None:
-            python_lib = sysconfig.get_python_lib(True, True,
-                                                  self.install_data)
-            pkgconfig_dir = os.path.join(os.path.dirname(python_lib),
-                                         'pkgconfig')
+            pkgconfig_dir = os.path.join(self.install_data, "lib64", 'pkgconfig')
         else:
             pkgconfig_dir = change_root(self.root, self.pkgconfigdir)
         self.mkpath(pkgconfig_dir)
